leanblueprint/jekyll_templates/_layouts/default.html

```diff
@@ -28,6 +28,7 @@ <h1 class="project-name">{{ page.title | default: site.title | default: site.git
     <h2 class="project-tagline">{{ page.description | default: site.description | default: site.github.project_tagline
       }}</h2>
     <a href="blueprint" class="btn">Blueprint (web)</a>
+    <a href="blueprint.pdf" class="btn">Blueprint (pdf)</a>
```

Comment:
should this link go to "blueprint/blueprint.pdf"?

Reply:
No, `"blueprint.pdf"` is correct. See [here](https://github.com/PatrickMassot/leanblueprint/blob/fe4bec2a72c69e9e2a1c674593a7d3e3c18eca2b/leanblueprint/templates/blueprint.yml#L81C42-L81C65).

Reply:
And I also tested on multiple projects: 

- [InfinityCosmos](https://emilyriehl.github.io/infinity-cosmos/)
- [ExpDB](https://teorth.github.io/expdb/)